Code/Legacy/CrySystem/AZCoreLogSink.h

```diff
@@ -31,6 +31,14 @@ class AZCoreLogSink
     : public AZ::Debug::TraceMessageBus::Handler
 {
 public:
+    ~AZCoreLogSink()
+    {
+        if (BusIsConnected())
+        {
+            Disconnect();
+        }
```

Comment:
It is always safe to unconditionally `Disconnect()` from an EBus.
It is the other way that causes Asserts. Connecting without checking whether the bus was connected.

Reply:
Fixed.